cmd/check_operator.go

```diff
@@ -60,7 +60,6 @@ var checkOperatorCmd = &cobra.Command{
 			os.O_WRONLY|os.O_CREATE|os.O_TRUNC,
 			0600,
 		)
-
```

Comment:
This does not appear to be relevant to this patch.

Reply:
Correct. No idea why it got in with the others.